src/ComScoreStreamingAnalytics.ts

```diff
@@ -82,10 +90,43 @@ export class ComScoreStreamingAnalytics {
     this.player.on(this.player.exports.PlayerEvent.StallStarted, this.stallStarted);
     this.player.on(this.player.exports.PlayerEvent.StallEnded, this.stallEnded);
     this.player.on(this.player.exports.PlayerEvent.AdBreakStarted, this.adBreakStarted);
+    this.player.on(this.player.exports.PlayerEvent.TimeChanged, this.timeChanged);
+  }
+
+  private blured = () => {
+    if (this.player.ads.isLinearAdActive()) {
+      this.stopComScoreTrackingBackgroundAd();
+      console.log('BLURED');
+    }
+  }
+
+  private timeChanged = (event: PlaybackEvent) => {
+    if ( ( this.playerTransitioningState === PlayerTransitionState.TransitioningToVideo ) && ( this.comScoreState !== ComScoreState.Video ) ) {
+      let rawData = this.rawData(this.player.getDuration());
+      this.streamingAnalytics.playVideoContentPart(rawData, this.contentType());
+      this.comScoreState = ComScoreState.Video;
+      this.logger.log('ComScoreStreamingAnalytics transitioned to Video - ' + JSON.stringify(rawData));
+      this.playerTransitioningState = PlayerTransitionState.Transitioned;
+    }
+    if ( ( this.playerTransitioningState === PlayerTransitionState.TransitioningToAd ) && (this.comScoreState !== ComScoreState.Advertisement) ) {
+      this.streamingAnalytics.playVideoAdvertisement({
+        ns_st_cl: this.currentAd.duration * 1000,
+        ns_st_ci: this.currentAd.id,
+        ns_st_st: '*null',
+        ns_st_ia: '*null',
+        ns_st_ddt: '*null',
+        ns_st_tdt: '*null',
+      }, this.adType());
+      this.comScoreState = ComScoreState.Advertisement;
+      this.logger.log('ComScoreStreamingAnalytics transitioned to Ad');
+      this.playerTransitioningState = PlayerTransitionState.Transitioned;
+    }
   }
 
   private paused = (event: PlaybackEvent) => {
-    this.stopComScoreTracking();
+    if (! this.player.ads.isLinearAdActive()) {
+      this.stopComScoreTracking();
```

Comment:
Why do we now just stop tracking if an ad is active? And no longer always when we pause?